src/components/network/network-table.js

```diff
@@ -238,6 +408,7 @@ const NetworkTable = (props) => {
                     props.network.threeWindingsTransformers[index]
                 }
                 columns={[
+                    makeHeaderCell('THREE_WINDING_TRANSFORMER'),
```

Comment:
WINDINGS (to be consistent with changes requested in gridsuite/network-modification-server#10)